app/src/main/java/org/dhis2/metadata/usecases/MapStyleConfiguration.kt

```diff
@@ -0,0 +1,46 @@
+package org.dhis2.metadata.usecases
+
+import org.dhis2.maps.layer.basemaps.BaseMapStyle
+import org.dhis2.maps.layer.basemaps.BaseMapStyleBuilder.build
+import org.hisp.dhis.android.core.D2
+import org.hisp.dhis.android.core.map.layer.MapLayerImageryProvider
+
+class MapStyleConfiguration(private val d2: D2) {
```

Comment:
shouldn't this class be on maps module?

Reply:
The map library has no knowledge of the sdk, that's why i have not include it.
But we could add the sdk dependency and move this class to the module if we think that is better